### DISCUSSION TITLE
Source build and rpm

### DISCUSSION THREAD

**Discussion by tontief** [posit-dev/positron#4610] [Q&A]
Hi, are there any plans to provide rpm packages (I'm a Fedora user). Furthermore, I could not find any detailed build instructions (even in the tarball), are they available anywhere?
Thanks.

**Comment by jennybc** [comment]
@lionel- I think you may be the best informed about this. Would you chime in here?

**Reply by lionel-** [reply]
Everything is now fixed on the Ark side and I believe @sharon-wang is working on positron releases for all platforms that workbench supports, including RPM-based distros:

- RedHat 8
- RedHat 9
- SUSE 15+

Hopefully one of the RHEL binaries will work on Fedora?

**Reply by sharon-wang** [reply]
Yes I believe the RPM should work for RHEL, SUSE and Fedora. If we can verify the RPM works for those platforms now that the Ark side is done (thank you!), we can re-enable publishing the RPM artifact. I think that verification will be done as part of QA-ing the Ark work.

**Reply by sharon-wang** [reply]
This issue https://github.com/posit-dev/positron/issues/3715 may be a good one to follow to keep tabs on Fedora support!

**Reply by tontief** [reply]
Just to explain my current state: I use it on Fedora by manually unpacking the .deb and executing from the right subfolder (works without problems, might not be optimal).

**Reply by sharon-wang** [reply]
@tontief Today's release is our first release with an RPM build available! https://github.com/posit-dev/positron/releases/tag/2024.10.0-13

If you take it for a spin, it'd be great to hear from you -- hopefully working without problems :)

**Reply by tontief** [reply]
Installs and works perfectly on Fedora 40 Kernel 6.10.11-200.fc40.x86_64
Thanks everyone!

**Reply by CodeCox** [reply]
Installed and ran on Fedora 40 KDE painlessly ... Thanks!

minor branding issue - vscode logo? (sorry :smile: )
![image](https://github.com/user-attachments/assets/d655c45f-3ae8-47e9-9706-a4e2d5e6c59a)

PS.  But more seriously, I did hit a wall when I tried running `Quarto: Preview` but that error is a saga for another day
(story starts with no Quarto rpm ... not the latest Quarto version, pixi env, etc).

**Reply by juliasilge** [reply]
Thanks for the report! I have opened https://github.com/posit-dev/positron/issues/4972 to get that icon updated.
